### PR TITLE
(TBV) fix: memory limit

### PR DESCRIPTION
**Pull request by YuriCat**
Does repeatedly checking memory cost much?

### CODE REVIEW

File: handyrl/train.py [hunk 1]
```diff
@@ -489,17 +489,14 @@ def feed_episodes(self, episodes):
...
-            self.flags.add('memory_over')
-
         self.trainer.episodes.extend([e for e in episodes if e is not None])
-        while len(self.trainer.episodes) > maximum_episodes:
+        while len(self.trainer.episodes) > self.args['maximum_episodes']:
+            self.trainer.episodes.popleft()
+
+        while psutil.virtual_memory().percent >= 95:
+            if 'memory_over' not in self.flags:
+                warnings.warn("memory usage %.1f%% with buffer size %d" % (mem_used_ratio * 100, len(self.trainer.episodes)))
```
Comment (ikki407) [inline handyrl/train.py:498]: Is `mem_used_ratio` defined anywhere?

Does there exist a way not to use `self.flags`? (I'm thinking...)

Comment (ikki407) [reply handyrl/train.py:498]: related?
https://stackoverflow.com/a/22661838/10809665

Comment (YuriCat) [reply handyrl/train.py:498]: Oops

**Comment by ikki407** [comment]
Thank you for your coding. #204 has been merged!

**Pull request closed without merging**